/home/groups/katrinjs/inputs//fasta_sequences/
/oak/stanford/groups/katrinjs/alphafold_data/
I0322 19:25:31.414714 140393224664896 templates.py:857] Using precomputed obsolete pdbs dummy_database//dummy_obsolete.dat.
I0322 19:25:32.732348 140393224664896 xla_bridge.py:170] Remote TPU is not linked into jax; skipping remote TPU.
I0322 19:25:32.732458 140393224664896 xla_bridge.py:355] Unable to initialize backend 'tpu_driver': Could not initialize backend 'tpu_driver'
I0322 19:25:32.732519 140393224664896 xla_bridge.py:355] Unable to initialize backend 'cuda': module 'jaxlib.xla_extension' has no attribute 'GpuAllocatorConfig'
I0322 19:25:32.732560 140393224664896 xla_bridge.py:355] Unable to initialize backend 'rocm': module 'jaxlib.xla_extension' has no attribute 'GpuAllocatorConfig'
I0322 19:25:32.732727 140393224664896 xla_bridge.py:355] Unable to initialize backend 'tpu': module 'jaxlib.xla_extension' has no attribute 'get_tpu_client'
I0322 19:25:32.732782 140393224664896 xla_bridge.py:355] Unable to initialize backend 'plugin': xla_extension has no attributes named get_plugin_device_client. Compile TensorFlow with //tensorflow/compiler/xla/python:enable_plugin_device set to true (defaults to false) to enable this.
W0322 19:25:32.732822 140393224664896 xla_bridge.py:362] No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
I0322 19:25:47.200030 140393224664896 run_alphafold.py:404] Have 25 models: ['model_1_multimer_v2_pred_0', 'model_1_multimer_v2_pred_1', 'model_1_multimer_v2_pred_2', 'model_1_multimer_v2_pred_3', 'model_1_multimer_v2_pred_4', 'model_2_multimer_v2_pred_0', 'model_2_multimer_v2_pred_1', 'model_2_multimer_v2_pred_2', 'model_2_multimer_v2_pred_3', 'model_2_multimer_v2_pred_4', 'model_3_multimer_v2_pred_0', 'model_3_multimer_v2_pred_1', 'model_3_multimer_v2_pred_2', 'model_3_multimer_v2_pred_3', 'model_3_multimer_v2_pred_4', 'model_4_multimer_v2_pred_0', 'model_4_multimer_v2_pred_1', 'model_4_multimer_v2_pred_2', 'model_4_multimer_v2_pred_3', 'model_4_multimer_v2_pred_4', 'model_5_multimer_v2_pred_0', 'model_5_multimer_v2_pred_1', 'model_5_multimer_v2_pred_2', 'model_5_multimer_v2_pred_3', 'model_5_multimer_v2_pred_4']
I0322 19:25:47.200188 140393224664896 run_alphafold.py:421] Using random seed 51140244952842358 for the data pipeline
I0322 19:25:47.200341 140393224664896 run_alphafold.py:164] Predicting fasta_sequences
Traceback (most recent call last):
  File "/home/groups/katrinjs/run-hpc-alphafold/ParallelFold/run_alphafold.py", line 450, in <module>
    app.run(main)
  File "/home/users/dkavi/miniconda3/envs/alphafold/lib/python3.9/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/home/users/dkavi/miniconda3/envs/alphafold/lib/python3.9/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/home/groups/katrinjs/run-hpc-alphafold/ParallelFold/run_alphafold.py", line 426, in main
    predict_structure(
  File "/home/groups/katrinjs/run-hpc-alphafold/ParallelFold/run_alphafold.py", line 175, in predict_structure
    feature_dict = data_pipeline.process(
  File "/home/groups/katrinjs/run-hpc-alphafold/ParallelFold/alphafold/data/pipeline_multimer.py", line 244, in process
    with open(input_fasta_path) as f:
IsADirectoryError: [Errno 21] Is a directory: '/home/groups/katrinjs/inputs//fasta_sequences/'
